db/migrations/20230103102111_create_order_status.sql

```diff
@@ -0,0 +1,9 @@
+-- migrate:up
+CREATE TABLE order_status (
+    id INT NOT NULL AUTO_INCREMENT,
+    order_status VARCHAR(255) NOT NULL,
```

Comment:
```suggestion
    status VARCHAR(255) NOT NULL,
```
status 혹은 name으로 컬럼명을 바꿔주세요~ 테이블이름이랑 중복되기 때문입니다